app-0-base-model/src/main/java/org/geovistory/toolbox/streams/base/model/processors/HasTypeProperty.java

```diff
@@ -32,8 +37,7 @@ public class HasTypeProperty {
     @Inject
     OutputTopicNames outputTopicNames;
 
-    public HasTypeProperty(AvroSerdes avroSerdes, BuilderSingleton builderSingleton, InputTopicNames inputTopicNames, OutputTopicNames outputTopicNames) {
```

Comment:
@joschne I removed the AvroSerdes parameter, as I think it's not used anymore with the use of "as"

Reply:
In the main code we use dependency injection (as normal with quarkus) – there this constructor is not needed.
In the tests we don't use dependency injection (as we do currently not use the default quarkus test setup). Therefore we instantiate the classes ourselves in the test setup. In the tests, ConfiguredAvroSerdes should be passed in instead of AvroSerdes. I changed the code accordingly in the next commit (a9b2d53).